docs/_posts/2017-06-13-react-v15.6.0.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "React v15.6.0"
+author: flarnie
+---
```

Comment:
I think you'll need to add yourself to https://github.com/facebook/react/blob/master/docs/_data/authors.yml for this to become a link.